docs/current/bare-metal.md

```diff
@@ -17,6 +17,280 @@
             </style>
 </head><article class="markdown-body">
 
-# Bare Metal setup
+# Getting Started on Bare Metal
+
+[Home](https://ciscocloud.github.io/vaquero-docs/) | [Docs Repo](https://github.com/CiscoCloud/vaquero-docs/tree/master)
+
+## Prerequisites
+
+This getting started guide focuses on getting up and running in a bare metal environment. Like the VM-based getting started guide, we'll again configure and run vaquero in "standalone mode" to reduce the overall footprint of the environment.
+
+To continue with the guide, please have the following available:
+
+1. One bare metal machine for running vaquero in standalone mode. See the next section on appropriate configuration.
+2. A second bare metal machine which must support PXE booting and IPMI (for power management)
+2. Both machines must be on the same network, and the network should be free of any existing DHCP service
+3. Access to [vaquero docker repository](
+https://hub.docker.com/r/thecsikos/vaquero/); this is currently a private docker repository
+
+## Configure the vaquero server/agent
+
+Similar to the VM-based approach, we'll configure the first server to run vaquero in "standalone mode" using Docker. For our purposes, we're running RHEL 7.4, but you should be able to use any OS that docker supports.
+
+Packages required to continue:
+
+* docker
+* git
+
+### Get the vaquero image
+
+Login to docker hub using credentials that have access to the vaquero docker repo:
+
+```
+# NOTE: this command will prompt you for your credentials
+docker login docker.io
+```
+
+and pull the latest vaquero docker image:
+
+```
+docker pull thecsikos/vaquero:latest
+```
+
+## Prepare vaquero source of truth (SoT) directory
+
+For our purposes, we'll be using the local directory mode for the vaquero server's SoT. You can do the same using a github repo, however you will need to make sure your server is routable to the internet and available to receive webhooks which is outside the scope of this getting started guide.
+
+### Initial directory structure
+
+We'll use /var/vaquero-demo, but feel free to use a different location. Take care to keep things consistent:
```

Comment:
I had a little trouble parsing this. Maybe consider, 
```In this example our SoT is in /var/vaquero-demo, if you use a different location be sure to update the following commands accordingly.```

Reply:
You're right....how would you feel if I just took it out. I think it's implied that if you start typing other paths YMMV :)